sn_cli/src/subcommands/wallet.rs

```diff
@@ -200,7 +200,7 @@ fn deposit(root_dir: &Path, read_from_stdin: bool, dbc: Option<String>) -> Resul
     wallet.try_load_deposits()?;
 
     let deposited =
-        sn_dbc::Token::from_nano(wallet.balance().as_nano() - previous_balance.as_nano());
+        sn_transfers::Nano::from_nano(wallet.balance().as_nano() - previous_balance.as_nano());
```

Comment:
`sn_transfers::Nano::from_nano`

should just be `new` or `from` i think? nano::from_nano is odd